app/models/accounting/quickbooks/transaction_builder.rb

```diff
@@ -62,8 +62,8 @@ def build_for_qb(transaction)
 
       private
 
-      def customer_reference(organization)
-        Customer.new(organization: organization, qb_connection: qb_connection).reference
+      def customer_reference(transaction)
+        transaction.customer.reference
```

Comment:
I think we want to know if/when this line breaks (this is how I realized interest txns had to be handled differently). That's because as far as I know, the madeline/qb relationship depends on the qb customer being set on txns going to qb, and this method is used on txns going to qb (not coming from qb). But let me know if that's not quite the right approach here.